src/inputs.rs

```diff
@@ -3,6 +3,7 @@ use std::thread;
 
 pub enum Input {
     LinuxTrackHead { yaw: f32, pitch: f32 },
+    #[cfg(features = "tobii-sys")]
```

Comment:
This one is different from the others, it's `features` instead of `feature` (I don't know if the plural form does the same thing or if it works) and also checks for `tobii-sys` instead of `tobii`.

Probably better to make it consistent, even if this works.

Reply:
good catch. for the record it still compiled without tobii, but i couldn't test with tobii so this might've broken that. maybe you can test compilation before merging?